py/server/deephaven/updateby.py

```diff
@@ -1190,10 +1190,12 @@ def rolling_count_time(ts_col: str, cols: Union[str, List[str]], rev_time: Union
 
 
 def rolling_std_tick(cols: Union[str, List[str]], rev_ticks: int, fwd_ticks: int = 0) -> UpdateByOperation:
-    """Creates a rolling standard deviation UpdateByOperation for the supplied column names, using ticks as the windowing unit. Ticks
-    are row counts, and you may specify the reverse and forward window in number of rows to include. The current row
-    is considered to belong to the reverse window but not the forward window. Also, negative values are allowed and
-    can be used to generate completely forward or completely reverse windows.
+    """Creates a rolling sample standard deviation UpdateByOperation for the supplied column names, using ticks as the
+    windowing unit. Sample standard deviation is computed using `Bessel's correction <https://en.wikipedia.org/wiki/Bessel%27s_correction>`_,
+    which ensures that the sample variance will be an unbiased estimator of population variance.
+    Ticks are row counts, and you may specify the reverse and forward window in number of rows to include.
+    The current row is considered to belong to the reverse window but not the forward window. Also, negative values are
+    allowed and can be used to generate completely forward or completely reverse windows.
```

Comment:
In all of these places, the bessel part should be a separate paragraph after the original paragraph.